src/libutil/error.hh

```diff
@@ -273,4 +273,28 @@ using NativeSysError =
  */
 void throwExceptionSelfCheck();
 
+/**
+ * Print a message and abort().
+ *
+ * @note: This assumes that the logger is operational
+ */
+[[noreturn]]
+void panic(std::string_view msg);
```

Comment:
I'd prefer a macro similar to `printError()` and `debug()` that wraps `fmt()` so we can write stuff like
```
panic("bla %s", arg);
```

Reply:
I was trying to keep it somewhat simple, because it might have to perform in a somewhat broken process state, though probably it shouldn't even use the logger then, and just `write` to fd `2`.

Reply:
I've made them signal safe.

For non-critical errors, we can throw `Error` as we did.